test_unstructured_ingest/test-ingest-sharepoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=Sharepoint
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+
+if [ -z "$MS_CLIENT_ID" ] || [ -z "$MS_CLIENT_CRED" ]; then
+   echo "Skipping Sharepoint ingest test because the MS_CLIENT_ID or MS_CLIENT_CRED env var is not set."
+   exit 0
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --ms-client-id "$SHAREPOINT_CLIENTID" \
+    --ms-client-cred "$SHAREPOINT_CRED" \
+    --ms-sharepoint-site "$SHAREPOINT_SITE" \
```

Comment:
these need to be passed in from CI. I can get the secrets loaded in, but then see the changes we do in `.github/workflows` in [this PR](https://github.com/Unstructured-IO/unstructured/pull/939/files) for reference for how these need to be passed through (both in ci and in the ingest-test-fixtures workflows)